File: baekjoon_3/2.py
#Python Code
num=int(input())
hl=[]
for i in range(num):
    x=map(int, input().split())
    x=list(x)
    hl.append(x)

for i in hl:
    res=0
    for a in i:
        res+=a
    print(res)

File: baekjoon_7/2.js
//N개의 숫자가 공백 없이 쓰여있다. 이 숫자를 모두 합해서 출력
var n = String(111111).split("");
console.log(n);
var res = 0;
for (var i = 0; i < n.length; i++) {
  res += Number(n[i]);
}
console.log(res);
